### ISSUE TITLE
obsolete comments about limitations on web pages

### ISSUE THREAD

**Issue by zookoatleastauthoritycom** [LeastAuthority/leastauthority.com#375]
Thanks to OliverUv on freenode, I've noticed two obsolete statements of limitation on https://leastauthority.com/how_it_works:

First:

"""
Are there limits on how much data I can store?

There are no limits on the total data you can store, but there is currently a limit on the size of an individual file. Files larger than about 3 GB might fail to upload. All upload failures, including failures caused by this, are shown in your LAFS client software, so you'll know if this happens to you.

We are working on lifting this limitation and will keep you informed about our progress.
"""

I *think* we long ago fixed that by deploying the cloud backend. I see that https://tahoe-lafs.org/trac/tahoe-lafs/ticket/1991 hasn't been closed yet, and I'm not sure why not. Is this limitation lifted?

Second:

"""
Can I delete old files to save space?

For now, it is not possible to remove data from the LAE
service. If you unlink a file, you lose your ability to access it,
but it does not free up the space in your S3 bucket, so you still
get charged for it.
"""

That's absolutely incorrect because customers no longer get charged per byte, but instead a flat fee. So rewrite this answer to:

"""
Can I delete old files to save space?

You can unlink a file, and if you didn't save any copies of the link to it anywhere, then it is effectively gone for good. You don't need to worry about saving space because Least Authority charges you only a flat monthly fee for your usage.
"""

**Comment by daira** [comment]
> I think we long ago fixed [the ~3GB immutable file limit] by deploying the cloud backend. I see that https://tahoe-lafs.org/trac/tahoe-lafs/ticket/1991 hasn't been closed yet, and I'm not sure why not. Is this limitation lifted?

No, this accurately reflects the current situation. Why did you think https://tahoe-lafs.org/trac/tahoe-lafs/ticket/1991 was fixed? The comments on that ticket are up-to-date.

When I read the email notification I didn't see that this was already a ticket, so I opened #376 for the second issue you mention. Let's leave this one for updating the FAQ after https://tahoe-lafs.org/trac/tahoe-lafs/ticket/1991 is fixed.
